html5ever/src/macros.rs

```diff
@@ -9,9 +9,9 @@
 
 macro_rules! unwrap_or_else {
     ($opt:expr, $else_block:block) => {
-        match $opt {
-            None => $else_block,
-            Some(x) => x,
+        {
+            let Some(x) = $opt else { $else_block };
+            x
```

Comment:
This change is necessary to minimize the scope of refcell borrows that are passed as arguments to the macro.